### ISSUE TITLE
[BUG] Red Borders on Some Empty Slots in Dragonflight Beta

### ISSUE THREAD

**Issue by Calesta** [arkayenro/arkinventory#1616]
**What version number you are using?**
v31000.DragonFlight.Alpha.11

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Dragonflight Beta

**What language is the game client set to?**
English

**Describe the bug**
Some of the empty slots have red borders.  I can still use the slots.

When I hover over the bags in the bag changer, I'm able to see that the empty slots with red borders are linked to different types of bags (e.g., Papa's Mint Condition Bag, Embroidered Deep Sea Bag) while other bags of the same type have empty slots that look 'normal'.

**To Reproduce**
Press 'B' to open the backpack.

**Screenshots**
![ArkInventory Red Squares](https://user-images.githubusercontent.com/13042214/196010892-5d540af2-1bf7-46f2-abef-31551b33b00e.jpg)

**Additional context**

**Comment by arkayenro** [comment]
blizzard have deprecated a whole bunch of `LE_ITEM_xxxxxx` globals that i will need to update to the equivalent `Enum.xxxxxx` values

**Comment by arkayenro** [comment]
should be fixed in 3.10.00 alpha 13

**Comment by Calesta** [comment]
I tried Alpha 13.

All my empty slots now have red borders and it seems that there are a lot more items that are being sorted into the proper bags (as you can see in the screenshot below when compared to the one above),

![WoWScrnShot_101822_230359](https://user-images.githubusercontent.com/13042214/196590006-174d3e00-d3c3-42c7-aa28-6929d7cf0d2e.jpg)

If you need any more detail, then please let me know. Thanks!

**Comment by arkayenro** [comment]
blizzard appears to have totally screwed up the bags - lots of people are reporting errors and issues without any mods
plus they seem to have  have changed some more stuff around so its broken the old code again

**Comment by arkayenro** [comment]
should be fixed in alpha 14, at least until blizzard break it again

**Comment by Calesta** [comment]
I tried Alpha 16.

The sorting seems much better (see below) but I'm still seeing the red borders.

![WoWScrnShot_102222_184055](https://user-images.githubusercontent.com/13042214/197365498-78dbabde-928e-4859-b859-e3c9ea43783d.jpg)

There *is* a lot of discussion about issues with bags/inventory on the beta right now so perhaps its related.  I'm just making sure I keep testing and reporting. :)

**Comment by arkayenro** [comment]
this was fixed but blizzard do keep breaking things.

what bag do the red borders belong to?  is it all of them?

**Comment by Calesta** [comment]
I have the Backpack, 3 Embroidered Deep Sea Bags, 1 Papa's Mint Condition Bag and 1 Simply Stitched Reagent Bag.  Any bag with an empty slot seems to have the red border.  

Something new is that now I can't even move things into the empty slots.  For example, I was trying to move something from my full Backpack to another bag to see if the red border affected the Backpack and I couldn't.  I tried other bags and the same thing happened.  It wasn't like that when I first reported this issue.
